neo_force_scheme/_neo_force_scheme.py

```diff
@@ -207,7 +207,7 @@ def transform(
             # randomly initialize the projection
             if starting_projection_mode == ProjectionMode.RANDOM:
                 # print(starting_projection_mode)
-                Xd = np.random.random((size, 2))
+                Xd = np.random.random((size, 3))
```

Comment:
The actual expectation is that the code can do both 2d and 3d. I suggest you add a parameter of transform with whether the user wants 2d or 3d (it can be something like n_dimensions).